samples/MvcSample.Web/wwwroot/Scripts/Mvc.js

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+var Mvc = (function ($) {
+
+    // This holds the stringified result.
+    var result = "";
+
+    // Takes the data which needs to be converted to MVC understandable format.
+    var _stringify = function (data) {
+        result = "";
+        for (var element in data) {
+            Process(element, data[element], null);
+        }
+
+        // An '&' is appended at the end. Removing it.
+        return result.slice(0, -1);
+    }
+
+    function Process(key, value, prefix) {
+        // Ignore functions.
+        if ($.isFunction(value)) {
```

Comment:
Could we use `typeof value === "function"` here? There's only one other jQuery use here which is used in a similar way - we could remove jQuery dependency from this piece of code.